app/controllers/api/v1/trips_controller.rb

```diff
@@ -1,18 +1,19 @@
 class Api::V1::TripsController < ApplicationController
+   before_action :set_trip, only: [:show, :update]
+   before_action :confirm_user, only: [:show, :update]
+   before_action :filter_user_trips, only: [:index]
+   
```

Comment:
What does the only: part do?

Reply:
I'm not sure I understand your question

Reply:
this means the filte_user_trips only work on index method, not all of the methods in the controller

Reply:
Ahhh, I see, thanks